ntp.go

```diff
@@ -150,11 +150,11 @@ func Offset(host string) (int64, error) {
 
 // offset variable names based off the rfc:
 // https://tools.ietf.org/html/rfc2030
-func offset(t1, t2, t3, t4 ntpTime) (int64, error) {
+func offset(clientSend, serverReceive, serverTransmit, clientReceive ntpTime) (int64, error) {
```

Comment:
Any reason why this is a separate helper function? Couldn't it be done directly in the Offset function?